grassroots-frontend/src/grassroots-shared

```diff
@@ -0,0 +1 @@
+/Users/ian/code/gpo/grassroots/grassroots-backend/src/grassroots-shared/
```

Comment:
This needs to be a relative path, but symlinks use absolute paths, right? Tricky.

Reply:
Once we dockerize, this can be an absolute path.

Reply:
Oops. Thanks for catching that! We should be able to make it a relative path.